Azure/azure-cli
# {PostgreSQL} Add tests for revive dropped for PG flexible servers

**Related command**
postgresql flexible-server revive-dropped

**Description**
Added test coverage for revive dropped functionality for PostgreSQL flexible servers

**Testing Guide**
Added automated test coverage for revive dropped functionality for PostgreSQL flexible servers

---

This checklist is used to make sure that common guidelines for a pull request are followed.
